### ISSUE TITLE
BasicUI: Presentation of nested elements

### ISSUE THREAD

**Issue by ThomDietrich** [eclipse/smarthome#2383]
Currently linked/nested elements (meaning elements which contain a group or a subframe) are presented with a ">" on the right **inline**. Example (please ignore the unicode tick symbol):

![2016-10-31 19_43_06-w66a openhab installation](https://cloud.githubusercontent.com/assets/2870104/19867353/d7584c9e-9fa3-11e6-8463-eafb29bb2011.png)

The special meaning of ">" is not clear and the alignment of the item text is confusing. For the sake of usability (and I'm far from being an expert in that field) I want to propose something along this idea:

![2016-10-31 19_43_06-w66a openhab installation - edit](https://cloud.githubusercontent.com/assets/2870104/19867518/6cfb6d80-9fa4-11e6-83cc-303ea1ab0f03.png)

**Comment by kaikreuzer** [comment]
Naaaah, I don't like that... It looks as if it will open a new browser window, which it definitely doesn't. Looking at my phone, the > seems to be a pretty natural choice:

![screen](https://cloud.githubusercontent.com/assets/3244965/19868519/b143885c-9fa8-11e6-88b1-465909ca8386.png)

So imho the only thing that can be discussed is the alignment (i.e. to decouple the text from it).

**Comment by MHerbst** [comment]
I agree with Kai, the ">" is more common to indicate an expandable element. But as one can see in Thom's  first screenshot it sometimes can be distinguished badly badly from the text.
I think it would be a good idea to add more space to the left of the ">" and maybe use a slightly larger font and and grey color.

**Comment by kaikreuzer** [comment]
@resetnow Want to have a try to optimize the appearance?

**Comment by ThomDietrich** [comment]
You are totally right. My choice was unfortunate. I just took the first symbol that came to mind. It is used to open a new window. I am **not** against the ">".  I am for a consistent alignment of the item text and for a better distinction of the symbol, for example by choosing a lighter color as can be seen in the iPhone screenshot. Moving the icon to the top right corner is also something that we can ignore when talking about the ">".

**Comment by resetnow** [comment]
Leaving a space for ">" wastes space for widget value which may not look as a problem on desktop but doesn't look good on mobile. 

One thing to do though would be aligning the ">" in line with other elements (it's not currently aligned).

![openhab1](https://cloud.githubusercontent.com/assets/2057191/19920604/c8e62166-a0ea-11e6-9c1f-65ef7ebbaadc.png)

Also, making it larger (something like 1.41 * line-height) probably would help. 

Couldn't find any material design guidelines in this regard. Also it would be good to hear an opinion of an actual UX/UI designer.

**Comment by ThomDietrich** [comment]
In Material Design there is no such thing. I think the rule is something along the line "Everything a user might click should be clickable" meaning if the fact if a menu element can be clicked should be relatable based on the content. This concept works in my android settings dialogs but I don't think it's a good idea for openHAB. Example: My room temperature element is nested, yours is not.

I do not think just aligning the ">" will do any good. From a UX perspective I would definitely expect the element text to be aligned and any additional UI elements to be in other positions. Your space argument is reasonable, however my screenshots represents the size on my smartphone screen and of course less space can be reserved for the ">".

**Comment by resetnow** [comment]
When less space is reserved for ">", it looks too close to the border and/or text and this is kind of ugly.
